### PR TITLE
fix: add logoHref and logoLabel props to footer

### PR DESCRIPTION
**Pull request by jorgemoya**
Add props for accessibility. Remove link when not being used.

### CODE REVIEW

**Review by migueloller — COMMENTED** [review]
What are the accessibility issues? It seems this component was simply unfinished. We should have the href as a prop as well as a label for it. It seems common to have a link back to the home page in the header ir footer, though

**Comment by jorgemoya** [comment]
> What are the accessibility issues? It seems this component was simply unfinished. We should have the href as a prop as well as a label for it. It seems common to have a link back to the home page in the header ir footer, though

Links need an accessible label.

I disagree regarding needing this link. Lets discuss offline.

**Comment by jorgemoya** [comment]
Updated PR to keep Link but don't use if not needed.

File: apps/web/vibes/soul/sections/footer/index.tsx [hunk 1]
```diff
@@ -74,22 +78,30 @@ export const Footer = forwardRef(function Footer(
...
                       <span className="whitespace-nowrap font-heading text-2xl font-semibold">
                         {logo}
                       </span>
                     </Link>
+                  ) : (
+                    <div className="relative mb-2 inline-block h-10 w-full max-w-56 rounded-lg ring-primary focus-visible:outline-0 focus-visible:ring-2">
+                      <span className="whitespace-nowrap font-heading text-2xl font-semibold">
+                        {logo}
+                      </span>
+                    </div>
```
Comment (migueloller) [inline apps/web/vibes/soul/sections/footer/index.tsx:96]: What if instead of duplicating the content we default the `href` to `#`?

Comment (jorgemoya) [reply apps/web/vibes/soul/sections/footer/index.tsx]: Fixed on latest commit.

**Review by migueloller — APPROVED** [review]

**Pull request merged**